urdf2webots/writeProto.py

```diff
@@ -397,93 +376,20 @@ def URDFVisual(proto, visualNode, level, normal=False):
         proto.write((shapeLevel + 2) * indent + 'radius ' + str(visualNode.geometry.sphere.radius) + '\n')
         proto.write((shapeLevel + 1) * indent + '}\n')
 
-    elif visualNode.geometry.trimesh.coord:
-        meshType = 'IndexedLineSet' if visualNode.geometry.lineset else 'IndexedFaceSet'
+    elif visualNode.geometry.mesh.url:
         if visualNode.geometry.defName is not None:
             proto.write((shapeLevel + 1) * indent + 'geometry USE %s\n' % visualNode.geometry.defName)
         else:
             if visualNode.geometry.name is not None:
                 visualNode.geometry.defName = computeDefName(visualNode.geometry.name)
             if visualNode.geometry.defName is not None:
-                proto.write((shapeLevel + 1) * indent + 'geometry DEF %s %s {\n' % (visualNode.geometry.defName, meshType))
-            else:
-                proto.write((shapeLevel + 1) * indent + 'geometry %s {\n' % meshType)
-            proto.write((shapeLevel + 2) * indent + 'coord Coordinate {\n')
-            proto.write((shapeLevel + 3) * indent + 'point [\n' + (shapeLevel + 4) * indent)
-            for value in visualNode.geometry.trimesh.coord:
-                proto.write('%lf %lf %lf, ' % (value[0] * visualNode.geometry.scale[0],
-                                               value[1] * visualNode.geometry.scale[1],
-                                               value[2] * visualNode.geometry.scale[2]))
-            proto.write('\n' + (shapeLevel + 3) * indent + ']\n')
-            proto.write((shapeLevel + 2) * indent + '}\n')
-
-            proto.write((shapeLevel + 2) * indent + 'coordIndex [\n' + (shapeLevel + 3) * indent)
-            if (isinstance(visualNode.geometry.trimesh.coordIndex[0], np.ndarray) or
-                    type(visualNode.geometry.trimesh.coordIndex[0]) == list):
-                for value in visualNode.geometry.trimesh.coordIndex:
-                    if len(value) == 3:
-                        proto.write('%d %d %d -1 ' % (value[0], value[1], value[2]))
-                    elif len(value) == 2:
-                        assert visualNode.geometry.lineset
-                        proto.write('%d %d -1 ' % (value[0], value[1]))
-            elif isinstance(visualNode.geometry.trimesh.coordIndex[0], np.int32):
-                for i in range(int(len(visualNode.geometry.trimesh.coordIndex) / 3)):
-                    proto.write('%d %d %d -1 ' % (visualNode.geometry.trimesh.coordIndex[3 * i + 0],
-                                                  visualNode.geometry.trimesh.coordIndex[3 * i + 1],
-                                                  visualNode.geometry.trimesh.coordIndex[3 * i + 2]))
+                proto.write((shapeLevel + 1) * indent + 'geometry DEF %s Mesh {\n' % (visualNode.geometry.defName))
```

Comment:
I believe these extra parenthesis are not needed:
```suggestion
                proto.write((shapeLevel + 1) * indent + 'geometry DEF %s Mesh {\n' % visualNode.geometry.defName)
```